.github/CHANGELOG.md

```diff
@@ -423,6 +423,29 @@
 
   In addition, the behaviour of `qml.math.requires_grad` has been improved in order to
   correctly determine trainability during Autograd and JAX backwards passes.
+
+* A new tape method, `tape.unwrap()` is added. This method is a context manager; inside the
+  context, the tapes parameters are unwrapped to NumPy arrays and floats, the the trainable
+  parameter indices are set.
+  [(#1491)](https://github.com/PennyLaneAI/pennylane/pull/1491)
+
+  These changes are temporary, and reverted on exiting the context.
+
+  ```pycon
+  >>> with tf.GradientTape():
+  ...     with qml.tape.QuantumTape() as tape:
+  ...         qml.RX(tf.Variable(0.1), wires=0)
+  ...         qml.RY(tf.constant(0.2), wires=0)
+  ...         qml.RZ(tf.Variable(0.3), wires=0)
+  ...     with UnwrapTape(tape) as unwrapped_tape:
```

Comment:
I overlooked it first time, but the explanation of the feature uses `tape.unwrap` and the example uses `UnwrapTape`...

Reply:
Oh, and do you want to advertise `Unwrap` here as well?

Reply:
done!